.golangci.yml

```diff
@@ -3,14 +3,27 @@ linters-settings:
     disable:
       - shadow # default value recommended by golangci
       - composites
+  
+linters:
+  enable:
+    - gosec
 
 run:
   build-tags:
     - integration
+  timeout: 5m
 
 issues:
   # Maximum issues count per one linter. Set to 0 to disable. Default is 50.
   max-issues-per-linter: 0
 
   # Maximum count of issues with the same text. Set to 0 to disable. Default is 3.
   max-same-issues: 0
+
+  exclude-rules:
```

Comment:
"excludes:" under linters-settings didn't work for some reason ([gosec config](https://golangci-lint.run/usage/linters/#gosec)). So i had to use text based exclusion

Reply:
I would honestly prefer doing local excludes over global excludes. Feels safer.